README.md

```diff
@@ -1,12 +1,26 @@
 # Vulcanomicon
 
-The Vulcan Academy's operating system
+The Vulcan Academy's operating system.
 
-1. The goal of the game is to create a learning organization
-1. All players must unanimously agree to all rule changes
-1. Proposals may add, amend or repeal a rule
-1. All rules should be logically self-consistent 
+1. The goal of the game is to create a learning organization.
+1. All players must unanimously agree to all rule changes.
+1. Proposals may add, amend or repeal a rule.
+1. All rules should be logically self-consistent. 
+
+## Reality
+1. Our understanding of __Reality__ is throttled by our ability to percieve it. 
+    * We use logic to describe our understanding of __Reality__.
+1. __Reality__ is exclusively inhabited by __Information__, and patterns thereof.
+    * __Information__ contained within __Empirical Reality__ inhabits 3 spatial dimensions and 1 monotonically increasing temporal dimension.
+1. __Information__ may interact with and modify other __Information__. This is an __Action__.
 
 ## Lexicon
 
-agent: any system that is best characterized as acting according to beliefs
\ No newline at end of file
+* __Reality__: The set of everything that exists or does not exist.
+* __Information__: What is conveyed by a particuar arrangement of things.
+* __Empirical Reality__: __Reality__ that is verifiable by observation rather than theory or pure logic.
+* __Action__: The interaction between __Information__.
+* __Decision Function__: Given __Reality__ _R_ and __Actions__ _A_ = { _a_<sub>0</sub>, ... }, we define __Decision Function__ _d_ such that _d_(_R_) = { _a_<sub>k</sub>, ... }.
+* __Agent__: A distinguishable set of __Information__ that has the ability to perform __Actions__. __Agents__ choose which __Action__ to perform based on a __Decision Function__.
```

Comment:
Maybe we could introduce a new term here:
Pattern: a distinguishable set of information

So then an Agent is a Pattern that has the ability to perform actions.